README.md

```diff
@@ -48,9 +48,43 @@ A quick note on environment variables
 
 Use `.env.local` for local development.
 
+### Server Side Environment Variables
+
+If a variable is required ONLY on the SERVER side (ie. in an api endpoint), then the variable can be declared in `.env.local.`
```

Comment:
All variables can/should be declared in `.env.local` for local development. For local development, we don't have the confusing issue about where env vars are coming from and how they are used since the app won't technically be built for local development.

Perhaps emphasize that `.env.local` is specifically a convention for local development and it's safe to put any and all env variable values there. This file is not committed to Github.